### ISSUE TITLE
having it *.github.io is useless

### ISSUE THREAD

**Issue by Wferr** [cstill/isitlatestart#1]
Just saying.

**Comment by cstill** [comment]
yeah I'm aware I'm gonna buy the domain and just give the repository a
cname file later

**Comment by Wferr** [comment]
How many users are you expecting to use the site? Not to be rude or anything, i don't think that many will use it.

**Comment by cstill** [comment]
honestly I hardly expect any. it's more of a learning thing for me and just
for personal convenience.

**Comment by Wferr** [comment]
If thats the case i would advise against buying a domain, would just be a waste of money.

**Comment by cstill** [comment]
good point. I'll probably just add it to a page on my regular site then

**Comment by Wferr** [comment]
Just rename the repo /lunch/ so it will appear under cstill.com/lunch/ or if your feeling fancy I can show you how to make it be under lunch.cstill.com
